packages/app-builder/src/components/Cases/CaseEvents.tsx

```diff
@@ -310,8 +329,16 @@ export function getEventDetail(event: CaseEvent) {
     case 'tags_updated': {
       return <TagsUpdatedEventDetail event={event} />;
     }
-    case 'decision_added':
+    case 'decision_added': {
+      if (!event.user_id) {
+        return <ByWorkflow type="added_by_workflow" />;
+      }
+      return <Author userId={event.user_id} type="added_by" />;
+    }
     case 'file_added': {
+      if (event.user_id === '') {
+        return <ByWorkflow type="added_by_workflow" />;
+      }
```

Comment:
Still `''` check here, probably to remove for this event

Reply:
oopsies, missed that one while shifting lines